_functions/compositions/ur.md

```diff
@@ -0,0 +1,51 @@
+---
+title: ur
+category: compositions
+---
+
+**Experimental**
+
+The `ur'` function is currently experimental and may change in future versions of Tidalcycles.
+
+~~~~ haskell
+ur' :: Time -> Pattern String -> [(String, Pattern a)] -> [(String, Pattern a -> Pattern a)] -> Pattern a
+~~~~
+
+`ur'` allows for the meta-composition of patterns. The first argument is a number of cycles
+that the `ur'` function loops at. The function will automatically split up the patterns defined
+across that number of cycles. The second argument is a pattern that defines how to compose
+the patterns to be defined. The third argument is a named array of Tidal patterns. The fourth argument is a named array of transition functions.
+
+~~~~ haskell
+d1 $ ur' 4 "[bd1 sn1]"
```

Comment:
May be, do you mean `ur`

Reply:
There is a function called `ur`, but we will be removing that function in an upcoming release.  We are favoring the usage of `ur'`.

Reply:
So, `ur'` will be for both:
``` haskell
ur :: Time -> Pattern String -> [Pattern a] -> Pattern a
```
and for
``` haskell
ur' :: Time
     -> Pattern String
     -> [(String, Pattern a)]
     -> [(String, Pattern a -> Pattern a)]
     -> Pattern a
```
??

Reply:
No.  The function definition for `ur` will not be available anymore.  `ur'` does everything that `ur` does plus more.